package Java_Programs.Strings;

import java.util.Scanner;

public class StringReverseWithoutFunction {

	public static void main(String[] args) {
			// TODO Auto-generated method stub
			Scanner sc=new Scanner(System.in);
			String s1=sc.nextLine();
			String s2="";
			for(int i=s1.length()-1;i>=0;i--)
				s2=s2+s1.charAt(i);
			System.out.println("Original String - "+s1+ "\nReversed String - "+ s2);
			sc.close();
	}

}
